test/react-native-cli/features/install.feature

```diff
@@ -0,0 +1,6 @@
+Feature: install command
+
+Scenario: running 'install' on an empty fixture
+    When I run the service "rn0_60" interactively
```

Comment:
Would be great if the RN version could be set by environment variable, then we'd have separate Buildkite jobs for each RN version, running in parallel.

Reply:
👍 done in https://github.com/bugsnag/bugsnag-js/pull/1153/commits/f2404dc65f60d7cc04af5a94584f1f71c619a7a6

Checked that this runs the 0.63 fixture:

```
$ REACT_NATIVE_VERSION=rn0_63 DEBUG=true docker-compose run --use-aliases react-native-cli
```

and this runs 0.60

```
$ REACT_NATIVE_VERSION=rn0_60 DEBUG=true docker-compose run --use-aliases react-native-cli
```